Amsterdam/cms
# Added imgproxy

To test:

1. Spin up docker
2. Get an image link from your local drupal (the link should look like: sites/default/files/inline-images/someimage.jpg)
3. Make it a base64 hash: (in js for example: `btoa('sites/default/files/inline-images/balcony.jpg')`) and copy the output
4. Go to this url with the base64 string: http://localhost:9091/assets/fill/1200/400/sm/0/<HERE_THE_BASE64>.jpg.
